src/pkg/packaging/dir.props

```diff
@@ -25,6 +25,7 @@
 
   <PropertyGroup>
     <CombinedInstallerFile>dotnet-$(ProductMoniker)$(CombinedInstallerExtension)</CombinedInstallerFile>
+    <CombinedInstallerEngine>dotnet-$(ProductMoniker)-engine.exe</CombinedInstallerEngine>
```

Comment:
I was looking before, and I don't see the below property `SharedFrameworkEngine` being used.  Can it be removed?